scripts/generate-analysis-files-for-release.sh

```diff
@@ -0,0 +1,123 @@
+
+# Fusion filtering
+echo "Create fusion filtered list"
+OPENPBTA_BASE_RELEASE=1 bash ${analyses_dir}/fusion_filtering/run_fusion_merged.sh
+
+# Fusion summary
+echo "Run fusion summary for subtypes"
+bash ${analyses_dir}/fusion-summary/run-new-analysis.sh
+
+# Compile all the files that need to be included in the release in one place
+# in the scratch directory
+compiled_dir=${scratch_dir}/analysis_files_for_release
+mkdir -p ${compiled_dir}
+
+# Copy over collapsed RNA-seq files
+cp ${analyses_dir}/collapse-rnaseq/results/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds ${compiled_dir}
+cp ${analyses_dir}/collapse-rnaseq/results/pbta-gene-expression-rsem-fpkm-collapsed.polya.rds ${compiled_dir}
+
+# Copy over independent specimen lists
+cp ${analyses_dir}/independent-samples/results/independent-specimens.*  ${compiled_dir}
+
+# Copy over fusions lists
+cp ${analyses_dir}/fusion_filtering/results/pbta-fusion-putative-oncogenic.tsv ${compiled_dir}
+cp ${analyses_dir}/fusion_filtering/results/pbta-fusion-recurrently-fused-genes-* ${compiled_dir}
+
+# Copy over fusion summary
+cp ${analyses_dir}/fusion-summary/results/* ${compiled_dir}
+
+# Run modules that cannot be run locally due to memory requirements
+if [ "$RUN_LOCAL" -lt "1" ]; then
+
+  # Run SNV consensus & TMB step for PBTA data
+  echo "Run SNV callers module for PBTA data"
+  OPENPBTA_BASE_RELEASE=1 bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-pbta.sh
+
+  # Run SNV consensus & TMB step for TCGA data
+  echo "Run SNV callers module for TCGA data"
+  bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-tcga.sh
+
+  # Copy over SNV callers
+  ## PBTA
+  cp ${analyses_dir}/snv-callers/results/consensus/pbta-snv-consensus-mutation.maf.tsv.gz ${compiled_dir}
+  cp ${analyses_dir}/snv-callers/results/consensus/pbta-snv-mutation-tmb-coding.tsv ${compiled_dir}
+  cp ${analyses_dir}/snv-callers/results/consensus/pbta-snv-mutation-tmb-all.tsv ${compiled_dir}
+  ## TCGA
+  cp ${analyses_dir}/snv-callers/results/consensus/tcga-snv-consensus-snv.maf.tsv.gz ${compiled_dir}
+  cp ${analyses_dir}/snv-callers/results/consensus/tcga-snv-mutation-tmb-coding.tsv ${compiled_dir}
+  cp ${analyses_dir}/snv-callers/results/consensus/tcga-snv-mutation-tmb-all.tsv ${compiled_dir}
+
+  # Run hotspot detection
+  echo "Run hotspots detection"
+  bash ${analyses_dir}/hotspots-detection/run_overlaps_hotspots.sh
+
+  # Copy over hotspots detection
+  cp ${analyses_dir}/hotspots-detection/results/pbta-snv-scavenged-hotspots.maf.tsv.gz ${compiled_dir}
+
+  # Run consensus CN caller step
+  echo "Run CNV consensus"
+  bash ${analyses_dir}/copy_number_consensus_call/run_consensus_call.sh
+
+  # Copy over CNV consensus
+  cp ${analyses_dir}/copy_number_consensus_call/results/pbta-cnv-consensus.seg.gz ${compiled_dir}
+
+  # Run GISTIC step -- only the part that generates ZIP file
+  echo "Run GISTIC"
+  # Run a step that subs ploidy for NA to allow GISTIC to run
```

Comment:
Just a question about the whether the CI if lines might be needed here.

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/e31b58ce6d6f01409a2fdef0191d6f8b6126a668/analyses/run-gistic/run-gistic-module.sh#L9-L32

Reply:
I don't follow. I don't necessarily expect this script (`scripts/generate-analysis-files-for-release.sh`) to get run in CI. 

We don't use `analyses/run-gistic/run-gistic-module.sh` at all here.

Reply:
> I don't necessarily expect this script (scripts/generate-analysis-files-for-release.sh) to get run in CI.

Just looking for a confirmation of this! 👍